packages/kbn-unified-field-list/src/containers/unified_field_list_sidebar/field_list_sidebar.scss

```diff
@@ -54,14 +54,12 @@
 .unifiedFieldListSidebar .unifiedFieldListItemButton {
   &.kbnFieldButton {
     margin-bottom: calc($euiSizeXS / 2);
-  }
-
-  &.domDragDrop-isDraggable {
-    box-shadow: none;
-  }
-
-  &:not(.unifiedFieldListItemButton__dragging) {
     padding: 0;
-    background: none;
   }
 }
+
+.unifiedFieldListItemButton__dragging {
+  padding: 0 $euiSizeXS;
+  border-radius: $euiBorderRadius;
```

Comment:
Maybe not related to this particular line, but the bottom corners on Lens field items look a bit off (not fully round):

<img width="269" alt="Screenshot 2024-03-12 at 13 25 24" src="https://github.com/elastic/kibana/assets/1415710/521df61f-3574-4c96-a9e6-e683ff762dae">

Reply:
Fixed!